docs/templates.md

```diff
@@ -18,6 +18,32 @@ In `jj log`/`jj obslog` templates, all 0-argument methods of [the `Commit`
 type](#commit-type) are available as keywords. For example, `commit_id` is
 equivalent to `self.commit_id()`.
 
+ * `description: String`
+ * `change_id: ChangeId`
+ * `commit_id: CommitId`
+ * `parents: List<Commit>`
+ * `author: Signature`
+ * `committer: Signature`
+ * `signature: CommitSignature`: The information about a cryptographic signature of the commit.
```

Comment:
I've moved the whole section under the "Commit type", so `signature: ..` will have to be moved there.

I'm also doing some structural changes in the template engine. It should be relatively easy to migrate to the new form, but auto-rebase will fail. Just fyi.